### DISCUSSION TITLE
Tablistrank Order

### DISCUSSION THREAD

**Discussion by AlhambraDev** [Zrips/CMI#6031] [Feature Requests]
Add a Option to set the order of the ranks in the tablist, like ghet the weigts from luckperms and set the order then from that or an extra option in config. So user dont have to install an extra plugin for the tablist order because cmi have a tablist feature but not the tablist order feature.

**Comment by AlhambraDev** [comment]
And the Rank and the name over the player would be nice for an extra option from the tablist :)

**Comment by CloudeLecaw** [comment]
one of the most often suggested features the last 3 years.
May it will come one day.

See itself
https://github.com/Zrips/CMI/discussions?discussions_q=tablist

For not it will land in discusstion thread line the other xx requests where player asked for tablist sorting

**Comment by Strahilchu** [comment]
I requested this before it just got pushed somewhere into enhancements and probably forgotten about forever again.

**Comment by CloudeLecaw** [comment]
> I requested this before it just got pushed somewhere into enhancements and probably forgotten about forever again.

This is not exacly correct...you requested this, i also request that and 10 others before you and me and after me the last 3 years.

This thread will be pushed to discussion aggain like the others the Last 3 years ;)


The Problem is that each time its landet in Suggestions area. Ppl start adding new threads instead of voting ans talking in the allready existing suggestions. This way it allways a bew thread mostly with a voting of 1. You can aspect that zrips wont take a look on new features that just have a voting of 1 in discusions (means just one man wants it each time, even the fakt is that many ppl want it.)

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending or descending by type, and types range from permission node weight to baltop rank, etc.
